### PR TITLE
Improve Sentry support

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Close https://github.com/enso-org/cloud-v2/issues/1590
  - Add source maps to built bundle for sentry to pick up
- Close https://github.com/enso-org/cloud-v2/issues/1589
  - Add global Sentry filter to avoid sending 400 responses from backend
- Wrap settings user and organization profile picture inputs in form
- Upgrade from deprecated sentry APIs

### Important Notes
None

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] ~~Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] ~~Unit tests have been written where possible.~~
- [x] ~~If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.~~
  - ~~If applicable, it is suggested to paste a link to a successful run of the Extra Tests.~~

### CODE REVIEW

**Review by MrFlashAccount — CHANGES_REQUESTED** [review]
Sentry doesn't pick up the sourcemaps: https://enso-zw.sentry.io/issues/6112647318/?project=4506070404300800&query=is:unresolved%20issue.priority:%5Bhigh,%20medium%5D&statsPeriod=24h&stream_index=0

image

**Comment by somebody1234** [comment]
@MrFlashAccount ready for re-CR i think

**Review by MrFlashAccount — APPROVED** [review]

**Comment by MrFlashAccount** [comment]
@somebody1234 let's merge it and see how it goes

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -701,10 +701,13 @@ jobs:
           ENSO_CLOUD_ENVIRONMENT: ${{ vars.ENSO_CLOUD_ENVIRONMENT }}
           ENSO_CLOUD_GOOGLE_ANALYTICS_TAG: ${{ vars.ENSO_CLOUD_GOOGLE_ANALYTICS_TAG }}
           ENSO_CLOUD_SENTRY_DSN: ${{ vars.ENSO_CLOUD_SENTRY_DSN }}
+          ENSO_CLOUD_SENTRY_ORGANIZATION: ${{ vars.ENSO_CLOUD_SENTRY_ORGANIZATION }}
+          ENSO_CLOUD_SENTRY_PROJECT: ${{ vars.ENSO_CLOUD_SENTRY_PROJECT }}
           ENSO_CLOUD_STRIPE_KEY: ${{ vars.ENSO_CLOUD_STRIPE_KEY }}
           ENSO_IDE_AG_GRID_LICENSE_KEY: ${{ vars.ENSO_AG_GRID_LICENSE_KEY }}
           ENSO_IDE_MAPBOX_API_TOKEN: ${{ vars.ENSO_MAPBOX_API_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SENTRY_AUTH_TOKEN: ${{ vars.SENTRY_AUTH_TOKEN }}
```
Comment (jdunkerley) [inline .github/workflows/release.yml]: this should be a secret not a var

File: app/gui/src/dashboard/index.tsx [hunk 2]
```diff
@@ -70,29 +78,29 @@ export interface DashboardProps extends app.AppProps {
...
- * Running this function finds a `div` element with the ID `dashboard`, and renders the
+ * Running this function finds a `div` element with the ID `enso-dashboard`, and renders the
  * authentication/dashboard UI using React. It also handles routing and other interactions (e.g.,
  * for redirecting the user to/from the login page).
  */
 export function run(props: DashboardProps) {
   const { vibrancy, supportsDeepLinks, queryClient, logger } = props
   if (
-    !detect.IS_DEV_MODE &&
+    // !detect.IS_DEV_MODE &&
```
Comment (jdunkerley) [inline app/gui/src/dashboard/index.tsx]: deliberately commented out?

**Review by jdunkerley — CHANGES_REQUESTED** [review]
The token should not be a variable it should be a secret.

**Comment by somebody1234** [comment]
@jdunkerley should be ready for re-review

**Review by jdunkerley — APPROVED** [review]
Thanks @somebody1234 - let me know what vars/secrets need added to org.

**Review by kazcw — APPROVED** [review]

**Pull request merged**